Lab_2_V1

Vonsec opened the pull request:
All tasks are performed using non-standard approaches.

Vonsec committed:
Lab_2_V1

ZebraHead22 commented on Lab2/Task_2.py:
18783.2758688244 - столько надо, чтобы выжить 10 месяцев...
Вы округлили, получили 18783. Хватит?..
Может надо округлить иначе?..
https://t.me/et_python2024/28

Vonsec replied:
I have corrected the code, now the value is rounded to a higher value.

Vonsec committed:
Update Task_2.py

Replaced round() with math.ceil() to round upwards. Now the value of the financial airbag is displayed correctly.
